src/Redmine/Api/Group.php

```diff
@@ -56,7 +59,7 @@ public function listing($forceUpdate = false)
      *
      * @param array $params the new group data
      *
-     * @throws \Exception Missing mandatory parameters
+     * @throws MissingParameterException Missing mandatory parameters
```

Comment:
now that the exceptions have meaningfull names, maybe those annotations are redundant, don't you think?

Reply:
We should keep the `@throws` annotations for better IDE integration.